### PR TITLE
Migrated Dried Kelp Block

### PR DESCRIPTION
**Pull request by RORIdev**

### CODE REVIEW

File: src/main/java/cn/nukkit/registry/BlockRegistry.java [hunk 1]
```diff
@@ -503,7 +503,7 @@ private void registerVanillaBlocks() {
         this.factoryMap.put(DARK_PRISMARINE_STAIRS, BlockStairsDarkPrismarine::new); //258
 
         this.factoryMap.put(PRISMARINE_BRICKS_STAIRS, BlockStairsPrismarineBricks::new); //259
-
+        this.factoryMap.put(DRIED_KELP_BLOCK,BlockDriedKelp::new); //UNK
```
Comment (joserobjr) [inline src/main/java/cn/nukkit/registry/BlockRegistry.java]: Please change `UNK` to `394`.

The block ID list can be seen here: https://minecraft.gamepedia.com/Bedrock_Edition_data_values#Block_IDs

**Comment by Gabriel8579** [comment]
Yes, I'll do it tonight

**Comment by RORIdev** [comment]
No items were unregistered. I've checked it.

**Comment by joserobjr** [comment]
Thank you!

**Pull request merged**